libcudacxx/test/libcudacxx/std/numerics/bit/bitops.count/countr_zero.pass.cpp

```diff
@@ -45,22 +45,29 @@ __host__ __device__ constexpr bool constexpr_test()
       && cuda::std::countr_zero(cuda::std::numeric_limits<T>::max()) == 0;
 }
 
+template <typename T>
+__host__ __device__ inline void assert_countr_zero(const volatile T val, int expected)
```

Comment:
The volatile here is not necessary. A function parameter should never be constexpr. Also can we call this just
```suggestion
__host__ __device__ inline void test(const T val, int expected)
```

Reply:
Compiler was being too tricky and constant folding the value. This seemed to trigger it.